doc/guides.md

```diff
@@ -15,7 +15,7 @@ It is possible to debug [Jest](https://facebook.github.io/jest/) tests in a Lern
     "runtimeArgs": [
         "exec",
         "--scope",
-        "my-package",
+        "@my-project/my-package",
```

Comment:
In this example, the package is named `my-package`, not `@my-project/my-package`. They need to match (the package name, not the directory name).

Reply:
true... i understand this... however having  a project fooproj with packages pkg1, pkg2, pkg3 

I had to run this with

```
runtimeArgs": [
        "exec",
        "--scope",
        "@fooproj/pkg1",
        "--",
        "node"
    ],
    "args": [
        "${workspaceRoot}/node_modules/jest/bin/jest.js",
        "--runInBand",
        "--no-cache",
        "packages/pkg1"
    ]
```

Please note `pkg1` is fine on the args but i had to specify the proj in the `runtimeArgs` ... i just want to reflect this in the guide. I'm not talking about directories... how do you suggest I can do that?

Reply:
This example is making an awful lot of assumptions about package structure and directory structure, true. The `--scope` argument requires the package name, and the Jest positional argument requires the directory path. There is no requirement with Lerna that the directory name matches the package name, so this is kinda... expected?

The current example is using a very common pattern (directory and package name match), but it is out of scope for this example to document [`--scope`](https://github.com/lerna/lerna/blob/master/core/filter-options/README.md#--scope-glob).